### ISSUE TITLE
Firefox Developer Edition keeps resetting prefenrences at any new tab

### ISSUE THREAD

**Issue by Yaro96** [YePpHa/YouTubeCenter#2201]
I have downloaded Firefox 46.0a2 64bit to be able to install the unsigned .xpi addon, the installation goes well, but at every new tab it resets the preferences of ytcenter (Build 537). When I import my ytcenter preferences it's all okay, but if I open any other video in new tab they are resetted, unless I open the video in the same tab.
The Greasemonkey userscript works well, though. So for now I'm forced to use that one.

**Comment by Pontiac76** [comment]
Maybe privacy mode?

**Comment by Yaro96** [comment]
Don't think so, all the cookies are stored, and I don't loose the login from youtube.

**Comment by ErdeFB** [comment]
Check that you don't have settings like not allowing 3rd party cookies and deleting stuff when you close the browser just in case.

**Comment by Yaro96** [comment]
Yes, I checked that, and it's all ok. As I said earlier the problem occurs even when I just open a new tab of youtube, there's no need to close the browser or open a new window or firefox.

**Comment by Yonezpt** [comment]
The add-on version is no longer compatible with the recent versions of Firefox https://github.com/YePpHa/YouTubeCenter/issues/2171#issuecomment-175823716

Use the userscript version instead.

**Comment by nazar-pc** [comment]
The problem is that extension itself works, but there is a but with preferences persistence.
I'm experiencing this for a long time now on Nightly and even disabled extension eventually because of this:(

**Comment by Yonezpt** [comment]
@nazar-pc Literally just one reply above yours I clearly said that the add-on version is no longer compatible with the recent versions of Firefox, use the userscript with Greasemonkey instead.

**Comment by nazar-pc** [comment]
I saw that, already installed it and it works fine
